### PR TITLE
Update : readme file on session secret

### PR DESCRIPTION
**Pull request by zsardone**

### CODE REVIEW

**Comment by zsardone** [comment]
@pdisabito can you please take a look at this update on readme file? Thank you!

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ To set up the sample application:
 <tr><td><code>COMMERCE_CLIENT_SHORT_CODE</code></td><td>Region-specific merchant identifier (for example, staging-001).</td></tr>
 <tr><td><code>COMMERCE_SESSION_SECRET</code></td><td>Some unique identifier for session management (for example, thisisasecretkey).</td></tr>
 </table>
+Note : The COMMERCE_SESSION_SECRET key needs to be unique per application, each customer should have a different key, it not then sessions could be bleed tetween ecommerce sites.
```
Comment (coopaq) [inline README.md]: typo "could be bleed tetween" should be "could be bleed between"

Comment (gurpreetsainisalesforce) [reply README.md]: typo "it not" should be "if not"

Comment (gurpreetsainisalesforce) [reply README.md]: > typo "could be bleed tetween" should be "could be bleed between"

"could bleed" sounds better?

**Comment by zsardone** [comment]
@pdisabito Your feedback is welcome on this small readme changes.

File: README.md [hunk 2]
```diff
@@ -30,11 +30,12 @@ To set up the sample application:
...
 <tr><td><code>COMMERCE_CLIENT_API_SITE_ID</code></td><td>A unique site ID (for example, RefArch or SiteGenesis).</td></tr>
 <tr><td><code>COMMERCE_CLIENT_CLIENT_ID</code></td><td>A unique ID used exclusively for API access. See <a href="https://documentation.b2c.commercecloud.salesforce.com/DOC1/topic/com.demandware.dochelp/AccountManager/AccountManagerAddAPIClientID.html">Add a Client API</a> for more information.</td></tr>
-<tr><td><code>COMMERCE_CLIENT_REALM_ID</code></td><td>A unique four-character ID (for example, bblx).</td></tr>
-<tr><td><code>COMMERCE_CLIENT_INSTANCE_ID</code></td><td>Instance ID within a realm (for example, 015).</td></tr>
-<tr><td><code>COMMERCE_CLIENT_SHORT_CODE</code></td><td>Region-specific merchant identifier (for example, staging-001).</td></tr>
-<tr><td><code>COMMERCE_SESSION_SECRET</code></td><td>Some unique identifier for session management (for example, thisisasecretkey).</td></tr>
+<tr><td><code>COMMERCE_CLIENT_REALM_ID</code></td><td>A unique four-character ID (for example, aaxy).</td></tr>
+<tr><td><code>COMMERCE_CLIENT_INSTANCE_ID</code></td><td>Instance ID within a realm (for example, 099).</td></tr>
```
Comment (echessman) [inline README.md]: Are we sure these are not real realm ID and realm instance ID?

Comment (zsardone) [reply README.md]: I am not sure these are real realm ID or will be in the future, so to be safe, I removed all the examples.

**Comment by syedshehrozhussain** [comment]
@zsardone can you rebase so that we can merge and close this? Changes look great here.

**Pull request merged**